### PR TITLE
update snapshot docs

### PR DESCRIPTION
**Pull request by dpprdan**
Some minor fixes to the snapshot.R docs:

- snapshots are saved as Markdown files, not JSON (right?)
- fix paths to snapshot folder
- typo

### CODE REVIEW

File: R/snapshot.R [hunk 1]
```diff
@@ -22,19 +22,19 @@
 #'
 #' @section Workflow:
 #' The first time that you run a snapshot expectation it will run `x`,
-#' capture the results, and record in `tests/testthat/snap/{test}.json`.
+#' capture the results, and record them in `tests/testthat/_snap/{test}.md`.
```
Comment (IndrajeetPatil) [inline R/snapshot.R:25]: Shouldn't this actually be `_snaps`? 
E.g., [here](https://github.com/r-lib/testthat/tree/main/tests/testthat/_snaps).

Comment (dpprdan) [reply R/snapshot.R]: 😆 good catch! Thank you @IndrajeetPatil

**Comment by hadley** [comment]
Fixes #1617

**Comment by hadley** [comment]
Thank you!

**Pull request merged**